usage/camp/pom.xml

```diff
@@ -186,6 +186,23 @@
     <build>
         <plugins>
 
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <configuration>
+                    <supportedProjectTypes>
+                        <supportedProjectType>jar</supportedProjectType>
+                    </supportedProjectTypes>
+                    <instructions>
+                        <Export-Package>org.apache.brooklyn.*</Export-Package>
+                        <Import-Package>
+                            !org.python.google.common.collect
```

Comment:
So far it seems to be because of winrm. Will be dropped once references to winrm are  contained in brooklyn-software-winrm, for instance.